src/bacnet/basic/service/s_dcc.c

```diff
@@ -56,7 +60,12 @@ uint8_t Send_Device_Communication_Control_Request(
     }
 
     /* is the device bound? */
-    status = address_get_by_device(device_id, &max_apdu, &dest);
+    status = address_get_by_device(
```

Comment:
this change is not needed if `address_segment_get_by_device()` is added to address.c module and address_get_by_device() remains unchanged